build.sh

```diff
@@ -75,10 +75,20 @@ function _build() {
           $git_fetch_url
 
   # 7. Tag and push the image to the ECR
-
-  docker tag $docker_registry_tag $docker_registry_tag
+  case $current_branch in main)
+    latest_tag=${docker_registry}:${component}-latest
+      ;;
+    *)
+      latest_tag=${docker_registry}:${component}-${current_branch}-latest
+      ;;
+  esac
+  docker tag $docker_registry_tag $latest_tag
+
+  p "Latest tag name (used for Cron jobs that don't know tag hashes on spin up..."
+  p $latest_tag
 
   p "Beginning push to ECR..."
+  docker push $latest_tag
```

Comment:
This looks fine, just what we talked about, but looking at it, I wonder if we want or need the non-main branches tagged as latest - we'll never use them and it may take up more ECR space.  🤔

Instead of the case with two branches, could we potentially put the latest bit in a conditional, something like this?

```
p "Push branch to ECR..."
docker push ${docker_registry_tag}

if [[ $current_branch == "main" ]]
    p "...and tagging main as latest for Cron jobs that don't know tag hashes on spin up"
    latest_tag=${docker_registry}:${component}-latest
    docker tag $docker_registry_tag $latest_tag
    docker push $latest_tag
fi
p "...push to ECR complete"
```